starter-code/src/memory.js

```diff
@@ -1,8 +1,37 @@
 class MemoryGame {
   constructor(card){
-    this.cards = cards;
+    this.cards = card;
+    this.pickedCards = [];
+    this.pairsClicked = 0;
+    this.pairsGuessed = 0;
+  }
+  shuffleCards() {
+    let currentIndex = this.cards.length, tempValue, randomIndex;
+
+    while (0 !== currentIndex) {
+      randomIndex = Math.floor(Math.random() * currentIndex--);
+      
+      tempValue = this.cards[currentIndex];
```

Comment:
Aqui vcs podem usar a sintaxe do ES6 para não precisar de uma variavel auxiliar:
 [this.cards[currentIndex],this.cards[randomIndex]] = [this.cards[randomIndex], this.cards[currentIndex]];